src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -688,12 +688,11 @@ private static DetailAST getFirstClassNameNode(DetailAST ast) {
     private static boolean isInIgnoreBlock(DetailAST methodBodyAst, DetailAST throwAst) {
         DetailAST ancestor = throwAst;
         while (ancestor != methodBodyAst) {
-            if (ancestor.getType() == TokenTypes.LITERAL_TRY
-                    && ancestor.findFirstToken(TokenTypes.LITERAL_CATCH) != null
-                    || ancestor.getType() == TokenTypes.LAMBDA
-                    || ancestor.getType() == TokenTypes.OBJBLOCK) {
-                // throw is inside a try block, and there is a catch block,
-                // or throw is inside a lambda expression/anonymous class/local class
+            if (ancestor.getType() == TokenTypes.LAMBDA
+                    || ancestor.getType() == TokenTypes.OBJBLOCK
+                    || ancestor.findFirstToken(TokenTypes.LITERAL_CATCH) != null) {
+                // throw is inside a lambda expression/anonymous class/local class,
+                // or throw is inside a try block, and there is a catch block
```

Comment:
`inside a try block, and there is a catch block` => `inside a catch block of a try`

Reply:
My understanding is that this checks if `throw` is inside the `try` block which has a succeeding `catch` block.
Wouldn't "`inside a catch block of a try`" mean that the `throw` _is not_ inside the `try` block but in the `catch` block of that `try` block?

Reply:
> **Unrelated:** I clicked on `re-request review` to signify that the review items were answered and were ready for review.
Please do let me know(for future cases) if I am not supposed to do that and reviewer(s) will check the PR again once they get time to do so.

Reply:
The point of the rewording was that the original seemed to equally mention try and catch when both tokens were written in the code. Now that one is removed, it should be more written about what is remaining which is in the catch block.

> mean that the throw is not inside the try block but in the catch block of that try block

I am not sure what this code is exactly referencing. So your saying the code is basically `throw is inside a block with a catch block`?

Reply:
While the try code is removed, the meaning of `ancestor.findFirstToken(TokenTypes.LITERAL_CATCH) != null` remains the same, as this being evaluated as `true` implicitly means that the type of ancestor is `LITERAL_TRY` (the reason why mutation was able to survive).

Now the literal meaning of this is that the `throw is inside an ancestor which has a child with the type "LITERAL_CATCH"`. --- (1)
But as we know that such ancestor can only be the type `LITERAL_CATCH`.

Do you think that the comment can be reworded as **(1)** or perhaps even better:
`throw is inside a try block, and that try block has a corresponding catch block`

Reply:
I am just going to resolve this to move on.